lib/JsonpMainTemplatePlugin.js

```diff
@@ -124,10 +124,11 @@ class JsonpMainTemplatePlugin {
 				var jsonpFunction = this.outputOptions.jsonpFunction;
 				return this.asString([
 					source,
+					"var __global = (0, eval)('this');",
```

Comment:
`eval` won't work because of CSP. I guess it's best to just allow to provide a option for the global object. In future we can use `global` as the spec adds it.

Reply:
@sokra , how about a new plugin as described in #5421 ?

Reply:
@sokra  How soon will be the `future`?

Reply:
Not that soon.

https://github.com/tc39/proposal-global